python/ray/dag/compiled_dag_node.py

```diff
@@ -1214,22 +1214,19 @@ def _get_or_compile(
             # or use InputAttributeNode, but not both.
             num_input_consumers = 0
 
-            # Step 1: populate num_channel_reads and perform some validation.
+            # Step 1: populate num_input_consumers and perform some validation.
```

Comment:
Thanks for catching this. `arg_to_consumers` should be the main thing, but you can mention `num_input_consumers` as well.

Reply:
update: https://github.com/ray-project/ray/pull/47489/commits/503351449a3d2462946fb2f899ba2d6eee789efd